libgnucash/engine/gnc-budget.cpp

```diff
@@ -683,35 +685,27 @@ get_perioddata (const GncBudget *budget, const Account *account, guint period_nu
     if (period_num >= priv->num_periods)
         throw std::out_of_range("period_num >= num_periods");
 
-    auto& map = priv->acct_map;
-    auto map_iter = map->find (account);
+    auto& vec = priv->acct_map->operator[](account);
```

Comment:
because `acct_map` is a `std::unique_ptr` whose contents are accessible via `->`. Alternatively it's possible to do `(*priv->acct_map)[account]` which is equally weird.